docs/mca/set-mca-param.rst

```diff
@@ -0,0 +1,251 @@
+
+.. danger:: Due to how the PMIx and PRRTE projects both evolved to
+            become independent projects from Open MPI (:ref:`see this
+            section for more detail
+            <label-running-role-of-pmix-and-prte>`), they both have
+            their own MCA system for setting MCA parameters.
+
+            Hence, all the information about MCA parameters below
+            *also* applies to PMIx and PRRTE.
+
+.. _label-mca-set-param-command-line:
+
+Command line parameters
+-----------------------
+
+The highest-precedence method is setting MCA parameters on the command
+line.  For example:
+
+.. code-block:: sh
+
+   shell$ mpirun --mca mpi_show_handle_leaks 1 -np 4 a.out
+
+This sets the MCA parameter ``mpi_show_handle_leaks`` to the value of
+1 before running ``a.out`` with four processes.  In general, the
+format used on the command line is ``--mca <param_name> <value>``.
+
+.. note:: When setting a value that includes spaces, you need to use
+          quotes to ensure that the shell understands that the
+          multiple tokens are a single value.  For example:
+
+          .. code-block:: sh
+
+             shell$ mpirun --mca param "value with multiple words" ...
+
+.. warning:: Setting Open MPI MCA parameters via the command line
+             entails using the ``--mca`` CLI option.  When setting
+             PMIx- and PRRTE-specific MCA parameters via configuration
+             files, use a different CLI option:
+
+             +----------+----------------+
+             | Open MPI | ``--mca``      |
+             +----------+----------------+
+             | PMIx     | ``--pmixmca``  |
+             +----------+----------------+
+             | PRRTE    | ``--prtemca``  |
+             +----------+----------------+
+
+.. _label-mca-set-param-env-var:
+
+Environment variables
+---------------------
+
+Next, environment variables are searched.  Any environment variable
+named ``OMPI_MCA_<param_name>`` will be used.  For example, the
+following has the same effect as the previous example (for sh-flavored
+shells):
+
+.. code-block:: sh
+
+   shell$ export OMPI_MCA_mpi_show_handle_leaks=1
+   shell$ mpirun -np 4 a.out
+
+.. note:: Just like with command line values, setting environment
+          variables to values with multiple words requires shell
+          quoting, such as:
+
+          .. code-block:: sh
+
+             shell$ export OMPI_MCA_param="value with multiple words"
+
+.. warning:: Setting Open MPI MCA parameters via environment variables
+             entails prefixing the parameter name with ``OMPI_MCA_``.
+             When setting PMIx- and PRRTE-specific MCA parameters via
+             environment variables, use a different prefix:
+
+             +----------+----------------+
+             | Open MPI | ``OMPI_MCA_``  |
+             +----------+----------------+
+             | PMIx     | ``PMIX_MCA_``  |
+             +----------+----------------+
+             | PRRTE    | ``PRRTE_MCA_`` |
```

Comment:
This isn't correct - it is `PRTE_MCA`. Only one R

Reply:
Fixed